src/util/experimentPreFetch.js

```diff
@@ -14,6 +14,7 @@ let activeExperiments = [
 	'pinned_filter',
 	'algolia_search',
 	'expandable_loan_cards',
+	'loan_res_20',
```

Comment:
global b/c it's used everywhere the AddToBasketInterstitial exists AND in Checkout